src/Client/ClientBase.cpp

```diff
@@ -3042,11 +3043,17 @@ void ClientBase::init(int argc, char ** argv)
         exit(0); // NOLINT(concurrency-mt-unsafe)
     }
 
+    if (options.count("verbose"))
+        config().setBool("verbose", true);
+
     /// Output of help message.
     if (options.count("help")
         || (options.count("host") && options["host"].as<std::string>() == "elp")) /// If user writes -help instead of --help.
     {
-        printHelpMessage(options_description);
+        if (config().getBool("verbose", false))
```

Comment:
I don't think we need to provide a default value (`false`).

Reply:
The config parameter current is available only when this parameter is used. That means we have two solutions:

1. set the value of this config to false if we cannot find the `verbose` in the list of used settings (not really common for our code)
2. leave the default value to false if we cannot find `verbose` in the list of used settings

Reply:
Okay, let's keep things then as is.